spec/models/workflow_step_spec.rb

```diff
@@ -34,6 +34,22 @@
       expect(subject.valid?).to be false
     end
   end
+  context 'without valid version number' do
+    it 'is not valid if the version number is nil' do
+      expect(subject.valid?).to be true
```

Comment:
why bother checking this?

Reply:
no particular reason - i suppose this is just double-confirming the existing rails test for their validation

Reply:
but that wouldn't fall under "without a valid version number".

Reply:
i guess i meant "version number" to generically mean the "version" attribute and not specifically a description of the data type